### PR TITLE
Rename BlockBody to ListBody.

### LINKED ISSUE #8888: Rename BlockBody and fix references to RenderBlockBase
I thought we had decided to rename BlockBody, but I can't find a bug on it anymore.

### PR DESCRIPTION
**Pull request by Hixie**
Nobody knew what a Block was.

Fixes https://github.com/flutter/flutter/issues/8888

### CODE REVIEW

File: packages/flutter/lib/src/widgets/basic.dart [hunk 1]
```diff
@@ -1539,19 +1539,21 @@ class SliverPadding extends SingleChildRenderObjectWidget {
...
-/// This widget is rarely used directly. Instead, consider using [SliverList],
+/// This widget is rarely used directly. Instead, consider using [ListView],
 /// which combines a similar layout algorithm with scrolling behavior, or
 /// [Column], which gives you more flexible control over the layout of a
 /// vertical set of boxes.
 ///
-/// For details about the block layout algorithm, see [RenderBlockBase].
-class BlockBody extends MultiChildRenderObjectWidget {
-  /// Creates a block layout widget.
+/// For more details about the layout algorithm, see [RenderListBody].
```
Comment (abarth) [inline packages/flutter/lib/src/widgets/basic.dart]: Cross reference ListView and SingleChildScrollView ?

**Comment by abarth** [comment]
LGTM

**Comment by abarth** [comment]
There are several of these in fuchsia.googlesource.com too.

**Comment by Hixie** [comment]
@apwilson I'll land this when it goes green, then land https://fuchsia-review.googlesource.com/#/c/25735/.

**Comment by abarth** [comment]
Probably worth announcing on flutter-dev

**Comment by Hixie** [comment]
Yeah I totally meant to. I sent an e-mail earlier today.

**Pull request merged**